docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>This code snippet (among others) shows that it's hard to ensure consistent formatting in our tutorials manually (cf. use of <code>'</code> and <code>"</code> ).</p><p> </p><p>I propose to include the newly added directories in the folder list passed to <a href="https://black.readthedocs.io/en/stable/" target="_blank">black</a> (i.e. <code>program_source</code> and <code>docs/tutorials</code>). Should be as easy as adding the folders to below lines and run <code>make black</code> afterwards.</p><ul><li>https://github.com/Qiskit/qiskit-ibm-runtime/blob/32607d677dbf570676701875f0cce38eef9fd5ef/Makefile#L24</li><li>https://github.com/Qiskit/qiskit-ibm-runtime/blob/32607d677dbf570676701875f0cce38eef9fd5ef/Makefile#L39</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done in <a href="https://github.com/Qiskit/qiskit-ibm-runtime/pull/97/commits/673f57c2567809016e0ff222ef1533807161a76f" target="_blank">673f57c</a></p>